File: ProjectEuler/Problem_2/sol.kt
var first = 1
var second = 2
var sum = 0
fun main(){

    while (second < 4_000_000){
        if (second % 2 == 0) {
            sum += second
        }

        second = second + first
        first = second - first
    }

    println(sum)
}

File: ProjectEuler/Problem_2/sol.ts
let sum:number = 0;

let first:number = 1;
let second:number = 2;
while (second < 1_00) {

    if (second % 2 == 0) {
        sum += second;
        // console.log(` first = ${first} second = ${second} `);
    }
    second = first + second;
    first = second - first;
}

console.log(sum);
